spell-ignore.txt

```diff
@@ -85,4 +85,6 @@ PieChart
 TextInput
 telerik
 TreeDataGrid
-Theming
\ No newline at end of file
+Theming
+bottomsheet
```

Comment:
Only BottomSheet should be added as this is the component's name. When you do not refer to the component, use "bottom sheet" as two separate words.
```suggestion
```